doc/configuration.md

```diff
@@ -247,29 +247,31 @@ appear in the container's /dev and allow read/write operations to it.
 
 The following properties exist:
 
-Key         | Type      | Default           | Required  | Description
-:--         | :--       | :--               | :--       | :--
-path        | string    | -                 | yes       | Path inside the container
-major       | int       | device on host    | no        | Device major number
-minor       | int       | device on host    | no        | Device minor number
-uid         | int       | 0                 | no        | UID of the device owner in the container
-gid         | int       | 0                 | no        | GID of the device owner in the container
-mode        | int       | 0660              | no        | Mode of the device in the container
+Key         | Type      | Default           | API extension                     | Required  | Description
+:--         | :--       | :--               |                                   | :--       | :--
+source      | string    | -                 |unix-block/unix-char_device_rename | no        | Path on the host
+path        | string    | -                 |unix-block/unix-char_device_rename | no        | Path inside the container(one of "source" and "path" must be set)
```

Comment:
Only "source" should have the API extension listed since "path" already existed before it.